.codespell-ignore-lines

```diff
@@ -16,3 +16,4 @@ Linix 45ZWN24-40           2  0.5 Ohm    0.400 mH  2.34A   24V
 * [#14927](https://github.com/apache/nuttx/pull/14927) spelling: fix spelling typo premption -> preemption
 * [#15520](https://github.com/apache/nuttx/pull/15520) drivers/note: fix typo falgs and align local name to irq_mask
         ans init
+	bool "IST HYT271 Humidity and Temperature Sensor support"
```

Comment:
should we add IST to white list

Reply:
yes, it is an alternative solution. Now I think its even better, because IST is also register name in x86 which probably also trigger codespell error.